assets/app/scripts/directives/resources.js

```diff
@@ -45,9 +53,28 @@ angular.module('openshiftConsole')
     return {
       restrict: 'E',
       scope: {
-        pods: '='
+        pods: '=',
+        projectName: '@?' //TODO optional for now
       },
-      templateUrl: 'views/_pods.html'
+      templateUrl: 'views/_pods.html',
+      controller: function($scope) {
+        $scope.phases = [
+          "Failed",
+          "Pending",
+          "Running",
+          "Succeeded",
+          "Unknown"
+        ];
+        $scope.expandedPhase = null;
+        $scope.warningsExpanded = false;
+        $scope.expandPhase = function(phase, warningsExpanded, $event) {
+          $scope.expandedPhase = phase;
```

Comment:
Should only be that deployment.  Each instance of a directive gets its own
scope.

On Thu, Sep 3, 2015 at 11:01 AM, Sam Padgett <notifications@github.com>
wrote:

> In assets/app/scripts/directives/resources.js
> <https://github.com/openshift/origin/pull/4486#discussion_r38655537>:
>
> >        },
> > -      templateUrl: 'views/_pods.html'
> > +      templateUrl: 'views/_pods.html',
> > +      controller: function($scope) {
> > +        $scope.phases = [
> > +          "Failed",
> > +          "Pending",
> > +          "Running",
> > +          "Succeeded",
> > +          "Unknown"
> > +        ];
> > +        $scope.expandedPhase = null;
> > +        $scope.warningsExpanded = false;
> > +        $scope.expandPhase = function(phase, warningsExpanded, $event) {
> > +          $scope.expandedPhase = phase;
>
> If you have multiple deployments with collapsed pods, does this only
> expand one or all?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4486/files#r38655537>.
>